Source/WTF/wtf/posix/FileSystemPOSIX.cpp

```diff
@@ -49,11 +49,25 @@
 #include <wtf/text/StringHash.h>
 #include <wtf/HashMap.h>
 
-#if OS(MORPHOS) || OS(AMIGAOS)
+#if OS(MORPHOS)
 #include <libraries/charsets.h>
 #include <proto/dos.h>
 #endif
 
+// TODO: This gives a conflict with struct WTF::Lock. 
```

Comment:
This probably answers my earlier question wrt. -D__USE_INLINE__ since "Lock" is a DOS function. We should discuss how to clean this up because I would prefer to use CPPFLAGS of -D__USE_INLINE__ ; but I know from experience of other projects that there can be conflicting symbols. Still...

Reply:
I wonder if taking advantage of the "AmigaOS" C++ namespace can help us out here. How that combined with __USE_INLINE__ I am not sure. But, that would allow us to split AmigaOS functions from functions with the same name within webkit and its other libraries.

Reply:
Resolving this. Very similar discussion above. To get all the information see this and the discussion above.